tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist=
+    py27,
+    py34,
+    py35
+
+[flake8]
+max-line-length= 100
+exclude= tests/*
+
+[testenv]
+commands=py.test --tb native {posargs:tests}
```

Comment:
Hmm, that's weird. I get `ERROR: file not found: tests`, as I would expect. There's no `tests` directory in the repo: what is it actually finding?